.gitlab/pipeline/zombienet/polkadot.yml

```diff
@@ -269,3 +269,14 @@ zombienet-polkadot-malus-0001-dispute-valid:
     - /home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh
       --local-dir="${LOCAL_DIR}/integrationtests"
       --test="0001-dispute-valid-block.zndsl"
+
+zombienet-polkadot-backing-disabling
+  extends:
+    - .zombienet-polkadot-common
+  rules:
+    - when: manual
+  variables:
+    ZOMBIE_PROVIDER: "k8s"
+  script:
+    - RUST_LOG=zombienet_orchestrator=debug,zombienet_provider=debug
+      cargo test --package zombienet-sdk-tests --lib -- tests::test_backing_disabling --exact --nocapture
```

Comment:
Can't we just use `cargo test` here without specifying a concrete tests so that we execute everything?

Reply:
Yes, we can but I need to check if all the test can run with the same environment.